docs/tutorials/01_electronic_structure.ipynb

```diff
@@ -108,159 +111,478 @@
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "problem.properties"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<qiskit_nature.second_q.properties.particle_number.ParticleNumber at 0x1528704623a0>"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "problem.properties.particle_number"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<qiskit_nature.second_q.properties.angular_momentum.AngularMomentum at 0x1528704621f0>"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "problem.properties.angular_momentum"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 14,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<qiskit_nature.second_q.properties.magnetization.Magnetization at 0x1528704622e0>"
+      ]
+     },
+     "execution_count": 14,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "problem.properties.magnetization"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 15,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "<qiskit_nature.second_q.properties.dipole_moment.ElectronicDipoleMoment at 0x152870462790>"
+      ]
+     },
+     "execution_count": 15,
+     "metadata": {},
+     "output_type": "execute_result"
     }
    ],
    "source": [
-    "qubit_converter = QubitConverter(mapper=ParityMapper(), two_qubit_reduction=True)\n",
-    "qubit_op = qubit_converter.convert(main_op, num_particles=es_problem.num_particles)\n",
-    "print(qubit_op)"
+    "problem.properties.electronic_dipole_moment"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "This time only 2 qubits are needed. "
+    "For more information about these properties, please refer to [their tutorial](09_properties.ipynb)."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Now that the Hamiltonian is ready, it can be used in a quantum algorithm to find information about the electronic structure of the corresponding molecule. Check out our tutorials on Ground State Calculation and Excited States Calculation to learn more about how to do that in Qiskit!"
+    "## Solving the `ElectronicStructureProblem`\n",
+    "\n",
+    "In the following, we will compute the ground-state of our problem instance. To learn more about the individual components that go into the `GroundStateSolver`, please refer to the corresponding tutorials."
```

Comment:
> To learn more about the individual components that go into the GroundStateSolver, please refer to the corresponding tutorials.

Does this add much - if it linked to some more specifically it might be helpful. For a new user figuring corresponding stuff may be more of a challenge

Reply:
I'll make a specific link :+1: